src/addons/metaservice/src/main/java/ee/ria/xroad/proxy/serverproxy/RestMetadataServiceHandlerImpl.java

```diff
@@ -240,6 +256,34 @@ private void handleGetOpenApi(ProxyMessage requestProxyMessage) throws IOExcepti
             restResponse.getHeaders().add(new BasicHeader(MimeUtils.HEADER_CONTENT_TYPE,
                     DEFAULT_GETOPENAPI_CONTENT_TYPE));
         }
+
+    }
+
+    private OpenapiDescriptionFiletype getFileType(RestResponse response, String serviceDescriptionURL)
```

Comment:
Signature should be getFileType(HttpResponse, URI)

Reply:
Parametertype changed accordingly